gcn/notices/km3net/test/medal_ranking_alert.schema.json

```diff
@@ -0,0 +1,142 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/km3net/test/medal_ranking_alert.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "unevaluatedProperties": false,
+  "title": "KM3NeT/Gold-Silver-Bronze Alerts",
+  "description": "Candidate neutrino observation reported from the KM3NeT online analysis pipeline.",
+  "allOf": [
+    {
+      "$ref": "../../core/Reporter.schema.json",
+      "description": "Instrument (ARCAxxx/ORCAxxx) issuing the alert. Mission=KM3NeT; Messenger=Neutrino."
+    },
+    {
+      "$ref": "../../core/Alert.schema.json",
+      "description": "Tense role (test/current observation), type (initial/update/retraction) and datetime of the alert generation"
+    },
+    {
+      "$ref": "../../core/Event.schema.json",
+      "description": "Event ID (eg 1A. Nb increment for each new alert, letter increment for update/retraction of the nb). Event_Name (KM3-yymmddL) if appropriate. "
+    },
+    {
+      "$ref": "#/$defs/space_time_coord",
+      "description": "Most probable Localization (Time, Position, Error and Skymap if available)."
+    },
+    {
+      "$ref": "../../core/Statistics.schema.json",
+      "description": "FAR and other statistic related to the event probability."
+    },
+    {
+      "$ref": "../../core/AdditionalInfo.schema.json",
+      "description": "Description of the pipeline used to produce the alert."
+    }
+  ],
+  "properties": {
+    "$schema": true,
+    "packet_type": {
+      "type": "number",
+      "description": "packet_type provided by GCN as for the old GCN format, associatied to notice_type/topic Gold=, Silver=, Bronze=."
```

Comment:
Is this associated with your pipeline? 
Or, GCN Classic? As we will not rely on the old system in the future.

@jracusin would be have packet type with new system?
KM3NeT had following concern over email:"Additionally, my collaboration has requested that we include a packet_type in our alerts. Would it be possible to provide us with three distinct packet_type values (one for each of the Gold, Silver, and Bronze alerts), similar to how it was done with the classical VOEvent messages?"

Reply:
As you stated, we would like to be able to give a packet_type in the notice, like it was done in GCN Classic. The number here are random for example, but will be updated with the accurate value if we have one.

Reply:
Hi @Vidushi-GitHub @jracusin would it be possible to have a packet_type even if our notices are not part of GCN classic ? 
I know it is not needed from a technical point of view with Kafka, but we want to provide this information for compatibility with pipelines already existing in experiments susceptible to receive our alerts.

Reply:
@VincentLuminy We'd rather not reference back to GCN Classic notice formats.  We can definitely find a way to list your classification of "Gold, Silver, Bronze", but "packet_type" isn't the most descriptive or intuitive keyword.  

You could use the [classification in the statistics core schema](https://github.com/nasa-gcn/gcn-schema/blob/main/gcn/notices/core/Statistics.schema.json), though that is designed to have a probabilistic classification for each choice as a dictionary.  

You could use "additionalInfo" like [IceCube](https://github.com/nasa-gcn/gcn-schema/blob/main/gcn/notices/icecube/test/gold_bronze_track_alerts.example.json). 

Or you could create a custom schema keyword like you have done, but maybe name it something more intuitive like "alert_criteria" or "alert_credibility".